docs/src/admin-manual/configuration.md

```diff
@@ -937,24 +930,16 @@ failures] or [system errors] during ingest, so they can be downloaded for
 inspection via the user interface by operators if desired. At least one of the
 sections below must be configured.
 
-#### Internal storage configuration - S3-like bucket or filesystem
+#### Internal storage configuration - S3-like bucket
```

Comment:
I think a filesystem directory would work for an internal storage location, but we haven't tested it.

Reply:
oh, lol - ok, will add this back i guess